agent/rpc/peering/service.go

```diff
@@ -581,16 +612,12 @@ func (s *Server) TrustBundleListByService(ctx context.Context, req *pbpeering.Tr
 }
 
 func (s *Server) getExistingOrCreateNewPeerID(peerName, partition string) (string, error) {
```

Comment:
Is this function used any more?

Reply:
yeah this wraps around the `getPeeringOrNil` for the `PeeringWrite` endpoint IIRC